ansible_runner/utils.py

```diff
@@ -101,6 +101,29 @@ def dump_artifacts(kwargs):
     if not os.path.exists(private_data_dir):
         raise ValueError('private_data_dir path is either invalid or does not exist')
 
+    if 'role' in kwargs:
+        role = {'name': kwargs.pop('role')}
+        if 'role_vars' in kwargs:
+            role['vars'] = kwargs.pop('role_vars')
+
+        play = [{'hosts': kwargs.pop('hosts', 'all'), 'roles': [role]}]
+
+        if kwargs.pop('role_skip_facts', False):
+            play[0]['gather_facts'] = False
+
+        kwargs['playbook'] = play
+
+        if 'envvars' not in kwargs:
+            kwargs['envvars'] = {}
+
+        roles_path = kwargs.pop('roles_path', None)
+        if not roles_path:
```

Comment:
If there is an if-else block, the if should contain the condition without negation.
Something like:
```
if roles_path:
    # append to roles_path
else:
    # create roles_path
```
Something very minor and that does not matter much, but can make code slightly more readable.